src/services/AdvancedTradingView/useAdvancedTradingView.hook.ts

```diff
@@ -166,14 +166,6 @@ export const useAdvancedTradingView = () => {
     }
 
     fetchTrades()
-
```

Comment:
why did we remove this code?

Reply:
It wasn't working as expected... We need a more elaborated solution for this